edx_sga/static/js/src/edx_sga.js

```diff
@@ -433,7 +433,7 @@ function StaffGradedAssignmentXBlock(runtime, element) {
     function loadjs(url) {
         $('<script>')
             .attr('type', 'text/javascript')
-            .attr('src', url)
+            .attr('src', window.baseUrl + url)
```

Comment:
Just in case `baseUrl` wasn't defined.

Sorry that I forgot this in the original PR, so please contribute this lower priority fix upstream as well.

```suggestion
            .attr('src', (window.baseUrl || '/static/') + url)
```